storage-model/src/test/java/no/unit/nva/publication/storage/model/daos/WithCristinIdentifierTest.java

```diff
@@ -0,0 +1,83 @@
+package no.unit.nva.publication.storage.model.daos;
+
+import com.amazonaws.services.dynamodbv2.model.AttributeValue;
+import com.amazonaws.services.dynamodbv2.model.ComparisonOperator;
+import com.amazonaws.services.dynamodbv2.model.Condition;
+import com.amazonaws.services.dynamodbv2.model.QueryRequest;
+import no.unit.nva.model.exceptions.InvalidIssnException;
+import no.unit.nva.publication.service.ResourcesDynamoDbLocalTest;
+import nva.commons.core.SingletonCollector;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.MethodSource;
+
+import java.net.MalformedURLException;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Stream;
+
+import static no.unit.nva.publication.storage.model.DatabaseConstants.RESOURCES_BY_CRISTIN_ID_INDEX_PARTITION_KEY_NAME;
+import static no.unit.nva.publication.storage.model.DatabaseConstants.RESOURCES_TABLE_NAME;
+import static no.unit.nva.publication.storage.model.DatabaseConstants.RESOURCE_BY_CRISTIN_ID_INDEX_NAME;
+import static no.unit.nva.publication.storage.model.daos.DaoUtils.sampleResourceDao;
+import static no.unit.nva.publication.storage.model.daos.DaoUtils.toPutItemRequest;
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.containsInAnyOrder;
+
+public class WithCristinIdentifierTest extends ResourcesDynamoDbLocalTest {
+
+    public void init() {
+        super.init();
+    }
+
+    @ParameterizedTest
+    @MethodSource("withPrimaryKeyInstancesProvider")
+    public void dynamoDbClientReturnsItemWithMatchingCristinId(List<ResourceDao> daos) {
+        init();
```

Comment:
Why do we need to call init() again? It is called automatically before each test

Reply:
It is not tagged with the @BeforeEach, so not calling it makes the test fail. Seeing that I am to add another test it would probably be advantageous to git it the BeforeEach and not calling it in either method